### PR TITLE
Feature/itinerary

### PR DESCRIPTION
**Pull request by derjubu**
Fix issue #3 

- See overview of destinations saved in local storage
- Go from itinerary to add new Destination

Deployment
https://capstone-rev-feature-it-yit7kn.herokuapp.com/

### CODE REVIEW

File: src/app/App.tsx [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
+import Itinerary from './pages/Itinerary/Itinerary';
 
 function App(): JSX.Element {
   return (
     <BrowserRouter>
       <Routes>
-        <Route path="/" element={<AddDestination />} />
+        <Route path="/" element={<Itinerary />} />
+        <Route path="/addDestination" element={<AddDestination />} />
         <Route
```
Comment (julialoeschel) [inline src/app/App.tsx:14]: looks like this is twice for `path="/"`?

File: src/app/pages/AddDestination/AddDestination.tsx [hunk 2]
```diff
@@ -2,39 +2,36 @@ import React from 'react';
...
+  const [startDate, setStartDate] = useLocalStorage(
     'startDate',
-    destination.startDate
+    newDestination.startDate
   );
-  const [endDate, setEndDate] = useLocalStorage<string>(
+  const [endDate, setEndDate] = useLocalStorage(
     'endDate',
-    destination.endDate
+    newDestination.startDate
```
Comment (julialoeschel) [inline src/app/pages/AddDestination/AddDestination.tsx]: shoud be     newDestination.`end`Date, right?

**Review by julialoeschel — COMMENTED** [review]
great!! found 2 small things.

File: src/app/App.tsx (hunk 1 above)
Comment (derjubu) [reply src/app/App.tsx:14]: I changed the path="/". The first line (red) is the old value, the second line (green) is the new value.

File: src/app/pages/AddDestination/AddDestination.tsx (hunk 2 above)
Comment (derjubu) [reply src/app/pages/AddDestination/AddDestination.tsx]: Yes! Copy/Paste error 🙄  Thanks for spotting!

**Pull request merged**